apps/site/lib/site_web/controllers/realtime_schedule_api.ex

```diff
@@ -0,0 +1,24 @@
+defmodule SiteWeb.RealtimeScheduleApi do
+  @moduledoc """
+    API for retrieving realtime schedule data
+  """
+  use SiteWeb, :controller
+
+  alias Site.RealtimeSchedule
+  alias Stops.Stop
+
+  def stops(conn, params) do
+    now = conn.assigns.date_time
```

Comment:
@ryan-mahoney I don't see where `conn.assigns.date_time` gets set in the codebase, though presumably it is if the tests pass...

Reply:
yeah, it is set here `apps/site/lib/site_web/plugs/date_time.ex`. This is a `Plug` which I'm generally against using for this reason... it's hard to tell when you are looking at a controller what data was added to the request context via previously called plug :/